recipes/libgsf/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "libgsf" %}
+{% set version = "1.14.46" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnome.org/pub/GNOME/sources/{{name}}/1.14/{{name}}-{{version}}.tar.xz
+  sha256: ea36959b1421fc8e72caa222f30ec3234d0ed95990e2bf28943a85f33eadad2d
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - perl
+    - perl-xml-parser
+    - pkg-config
+    - {{ compiler('c') }}
+  host:
+    - glib
+    - libxml2
+    - zlib
+    - bzip2
+    - pcre 8.42*
```

Should it be just ``8.42`` or maybe greater than?
```suggestion
    - pcre 8.42
```

---

All other pcre versions available gave me problems with local circleci testing.

---

I see, could you please pin it then? Just remove the ``*`` and it should be fine
:)

---

ah, you already did it!
:)